components/survey/List/index.jsx

```diff
@@ -82,17 +79,21 @@ const SurveyList = ({ setBackgroundImagePath }) => {
     }
   };
 
+  const sliderEvents = {
+    beforeChange: (_oldIndex, newIndex) => beforeSlideChange(newIndex),
+  };
+
   const sliderConfig = { ...sliderSettings, dotsClass: `slick-dots ${styles.slickDots}`, ...sliderEvents };
 
   return (
     <div className="w-[313px] md:w-[419px] lg:w-[704px] m-auto">
       <div className="font-extrabold text-white text-base-xs uppercase mb-1" data-test-id={surveyListTestIds.date}>
         {formatDate(date)}
       </div>
-      <div className="font-extrabold text-white text-base-xxxl mb-8" data-test-id={surveyListTestIds.text}>
+      <div className="font-extrabold text-white text-base-xxxxl mb-8" data-test-id={surveyListTestIds.text}>
         Today
       </div>
-      <Slider {...sliderConfig}>{surveys.map(survey => renderSurveyCard(survey))}</Slider>
+      <Slider {...sliderConfig}>{surveys.map(survey => renderSurveyCard(survey))}</Slider>;
```

Comment:
```suggestion
      <Slider {...sliderConfig}>{surveys.map(survey => renderSurveyCard(survey))}</Slider>
```

Reply:
done in 0a8596544c05464d0eef88743810267b02504566!